docs/synapse/userguides/storm_ref_cmd.ipynb

```diff
@@ -437,9 +472,101 @@
     "\n",
     "- ``delnode`` operates on the output of a previous Storm query.\n",
     "- ``delnode`` will attempt to perform some basic sanity-checking to help prevent egregious mistakes. For example, ``delnode`` will return an error if you attempt to delete a node that is still referenced by another node (such as an ``inet:fqdn`` that is referenced by an ``inet:dns:a`` node). Similarly, delnode will return an error if you attempt to delete a ``syn:tag`` node if that tag is still applied to other nodes. **However, delnode cannot prevent all mistakes.**\n",
+    "  \n",
+    "  - Edge / digraph nodes (such as ``edge:refs`` or ``edge:has``) are **exempt** from these \"reference checks\", It is possible to delete an edge node where ``n1`` and / or ``n2`` still exist and without affecting ``n1`` or ``n2``. This is generally desired behavior as the relationship between the nodes (the edge) may need to be removed without affecting the nodes themselves.\n",
```

Comment:
I think the comma at "from these \"reference checks\", " should be a period, assuming I'm reading it correctly.